recipe/meta.yaml

```diff
@@ -26,41 +25,44 @@ outputs:
       run:
         - python
         - soupsieve >=1.2
+      run_constrained:
+        - html5lib
+        - lxml
```

Comment:
AFAIU there's no point in specifying packages in `run_constrained` without pinning: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=run_constrained#run-constrained

Reply:
I think you would just judge whether you want these packages or not (should have a good reason to include them if so) and if you're including them, just put them in the run section

Reply:
I'll remove it. The size of the package matters